src/github/branch.ts

```diff
@@ -85,14 +85,14 @@ export const deleteBranchWebhookHandler = async (context: WebhookContext<DeleteE
 		return;
 	}
 
-	context.log.info({ prRef: context.payload.ref }, `Deleting branch for repo ${context.payload.repository?.id}`);
+	context.log.info({ prRef: context.payload.ref }, `Deleting branch for repo ${context.payload.repository.id}`);
```

Comment:
/nit <-- can ignore this comment.

Since the payload comes from external (and possible comes from multiple version of github cloud/server), and this use case is just a log, I'd say leave it as `repository?.id` for safety reason. 

Ideally, I'd suggest we should validate all external input with json schema (either directly download from github or generated from typings) before processing it.

Reply:
This is auto fixed by https://typescript-eslint.io/rules/no-unnecessary-condition/

The payload is a DeleteEvent that has Repository as a required member.  Repository should never be null here